spec/models/user_spec.rb

```diff
@@ -10,4 +10,16 @@
   it 'user full name should be correct' do
     expect(user.full_name).to eq('Winston Churchill')
   end
+
+  it "admin? method should be correct" do
+    expect(user.admin?).to be false
+  end
+
+  it "super_admin? method should be correct" do
```

Comment:
all is good in PR, but 2 tests super_admin and admin are bad.
read our chat and find something else to test

Reply:
commented